src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -830,8 +830,10 @@ The installation package contains 4 files:
 
 - The `inputConfig.json` file that includes the required parameters needed for the configuration tuning of the ProActive server. It is formatted as a json file.
 - The `proactiveInstaller.sh` file, which is the main installation script. It parses the configuration provided in the `inputConfig.json` file, proceeds to the configuration, and finally starts up the ProActive server.
+- The `password-template.properties` file contains the clear-text passwords required for installation, which will be encrypted during the process. This file is used only for a specific installation method (more details can be found here: <<_shell_Starting_ProActive_using_Shell_scripts, Starting ProActive using Shell scripts>>).
 - The `apiHealthcheck.sh`, called by the `proactiveInstaller.sh` (at the end), to run REST API health check tests to verify the correct behavior of the ProActive server after its startup.
```

Comment:
To ensure security, this file will be deleted once the installation is complete ?

Reply:
It will not be deleted automatically, I already provided in the installation instructions this note:

NOTE: After the installation is complete, it is recommended to delete the clear-text password file to enhance security.